bin/install-wp-tests.sh

```diff
@@ -188,7 +188,7 @@ install_deps() {
 	cd "$WP_CORE_DIR"
 	php wp-cli.phar plugin activate woocommerce
 
-	if [ "$TRAVIS_PULL_REQUEST_BRANCH" != "" ]; then
+	if [ "$TRAVIS_PULL_REQUEST_BRANCH" ]; then
```

Comment:
I'm not sure the syntax, but should have an AND statement?

```
if [ "$TRAVIS_PULL_REQUEST_BRANCH" && "$TRAVIS_PULL_REQUEST_BRANCH" != "" ];
```

Reply:
It doesn't need an `AND` - it essentially turns into either `if [ "" ]; then` for pushes to `master`, and `if [ "name/of-the-branch" ]; then` for pull requests.

Reply:
You can see what the problem is here: https://travis-ci.org/woocommerce/woocommerce-admin/jobs/588609339#L567